CMakeLists.txt

```diff
@@ -5,8 +5,7 @@ include("common.cmake")
 # Haywire
 # ----------------------------------------
 project(haywire C)
-set(CMAKE_BUILD_TYPE RelWithDebInfo)
-#set(CMAKE_BUILD_TYPE Debug)
+set(CMAKE_BUILD_TYPE Release)
```

Comment:
Here's the difference when it comes to compilation flags when using `Release` and `RelWithDebInfo``:

```
//Flags used by the compiler during release builds.
CMAKE_C_FLAGS_RELEASE:STRING=-O3 -DNDEBUG

//Flags used by the compiler during release builds with debug info.
CMAKE_C_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG
```

We're already setting `-O3`, so the only other difference is the usage of `-g`, which enables debug symbols to be embedded in the executable. It won't change the code paths, but it will make the executable a bit larger and may or may not cause additional page faults. I guess that's not a big deal given that Haywire is relatively small though.

I'm happy to revert the change if you prefer to have debug symbols by default.